src/LinuxTracing/SampleProgram.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <string.h>
+
+#include <atomic>
+#include <chrono>
+#include <iostream>
+#include <mutex>
+#include <thread>
+
+static std::atomic<bool> KEEP_GOING;
+static std::atomic<int> num;
+
+std::mutex mtx;
+
+constexpr int kNumOfThreads = 4;
```

Comment:
I would move this declaration down to right before it's used.

Reply:
This line (`constexpr int kNumOfThreads = 4;`) should be the first line in the `main` function

Reply:
This is irrelevant after the new changes.